chips/internal/_shared.scss

```diff
@@ -78,6 +79,12 @@
     pointer-events: none;
   }
 
+  .container.has-icon {
+    .label-container {
```

Comment:
Can you collapse these lines since google Sass style guide allows nesting only if it prevents repetition